### PR TITLE
feat: save username to localstorage

### PR DESCRIPTION
**Pull request by me22mer**
image

### CODE REVIEW

**Comment by viruskizz** [comment]
single value is not dynamic support. there are attributes more than for user. 
It better to save as object.

**Comment by viruskizz** [comment]
User service to save value to `localstorage.service.ts`

**Comment by me22mer** [comment]
@viruskizz 
image

File: frontend/src/app/app.routes.ts [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
 import { HomeComponent } from './features/home/home.component';
+import { NameInputComponent } from './features/name-input/name-input.component';
 
 export const routes: Routes = [
   {
     path: '',
     component: HomeComponent,
   },
+  {
+    path: 'name-input',
```
Comment (viruskizz) [inline frontend/src/app/app.routes.ts:11]: Please use prefix test for test component

File: frontend/src/app/app.routes.ts [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
 import { HomeComponent } from './features/home/home.component';
+import { NameInputComponent } from './features/name-input/name-input.component';
 
 export const routes: Routes = [
   {
     path: '',
     component: HomeComponent,
   },
+  {
+    path: 'test-name-input',
```
Comment (viruskizz) [inline frontend/src/app/app.routes.ts:11]: path should be `/tests/...`

**Review by viruskizz — APPROVED** [review]

**Pull request merged**